tools/dds/dds-sniffer/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+#  minimum required cmake version: 3.1.0
+cmake_minimum_required(VERSION 3.1.0)
+
+if(BUILD_WITH_DDS)
+    project(Realsensetoolsdds-sniffer)
```

Comment:
If you change the project name to something like:
project(rs-dds-sniffer) 
This way you can replace all references to it with  `${PROJECT_NAME}`

Like this:
```
add_executable(${PROJECT_NAME} dds-sniffer.cpp)
set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
```
and so on..

Reply:
Right -- and, following RS convention, the executable name (like Nir mentioned) should be `rs-dds-sniffer`